src/instructlab/configuration.py

```diff
@@ -248,10 +254,9 @@ class _general(BaseModel):
 
     # model configuration
     model_config = ConfigDict(extra="ignore")
-
     # additional fields with defaults
-    log_level: StrictStr = "INFO"
-    debug_level: int = 0
+    log_level: StrictStr = Field(default="INFO", description="Log level for logging.")
+    debug_level: int = Field(default=0, description="Debug level for logging.")
```

Comment:
Should we be describing what these values are? I feel like this can be inferred from the names. It may be a better use of real estate to include what the valid values are, e.g. "debug", "info", etc.

It may also be helpful to differentiate between what log level & debug level are.

Reply:
I don't want to change the current behavior. I'm simply converting to Field and put a description. Can we address this later?